modules/hello-world/pages/start-using-sdk.adoc

```diff
@@ -439,19 +437,19 @@ Like `replace`, `remove` also optionally takes the CAS value if you want to make
 == Data Modeling
 
 Documents are organized into collections -- collections of documents that belong together.
-You get to decide what it means to "belong." 
+You get to decide what it means to "belong". 
 Developers usually put documents of the same type in the same collection.
 
-For example, imagine you have two types of documents: customers and invoices. 
-You could put the customer documents in a collection called `customers`, and the invoice documents in a collection called `invoices`.
+For example, consider you have two types of documents: customers and invoices. 
+You can put the customer documents in a collection named `customers`, and the invoice documents in a collection named `invoices`.
 
-Each document belongs to exactly one collection. 
+Each document belongs to one collection. 
 A document's ID is unique _within_ the collection.
 
-Different scopes can hold collections with different names. 
+Different scopes hold collections with different names. 
 There is no relationship between collections in different scopes.
 Each collection belongs to just one scope and
-a collection's name is unique within the scope.
+the collection name is unique within the scope.
 
 
 More details can be found on the xref:concept-docs:data-model.adoc[Data Model page].
```

Comment:
A somewhat optimistic statement on my part - that linked page is one that's worth a bit of time improving.